charts/rancher-monitoring/v0.2.1/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.4-rc1
```

Comment:
Shouldn't this be 2.5.6-rc1 and the monitoring v0.2.0 should have a max Rancher version so only 1 Rancher version is available for this line of monitoring?